src/puripuly_heart/providers/stt/sixtydb.py

```
@@ -0,0 +1,351 @@
+
+        import websockets
+
+        async def _check() -> bool:
+            url = _build_ws_url(endpoint, api_key)
+            try:
+                async with websockets.connect(
+                    url, ping_interval=None, open_timeout=5
+                ) as ws:
+                    try:
+                        message = await asyncio.wait_for(ws.recv(), timeout=3.0)
+                    except asyncio.TimeoutError:
```

Ruff (0.15.15)

[warning] Replace aliased errors with `TimeoutError`

Replace `asyncio.TimeoutError` with builtin `TimeoutError`

(UP041)